.github/workflows/compile.yml

```diff
@@ -32,6 +32,12 @@ jobs:
 
       - run: nimble build -d:debug -Y -o:native_main
 
+
+      - run: |
+          sudo apt-get install gcc-arm-linux-gnueabihf
+          nimble build -d:debug -Y --cpu:arm -o:native_main_arm
```

Comment:
this sort of works but `-o` doesn't actually change the output, so the executable gets overwritten. Also I don't think `armhf` is what #37 was asking for